sql/core/src/test/scala/org/apache/spark/sql/jdbc/JDBCSuite.scala

```diff
@@ -56,6 +56,17 @@ class JDBCSuite extends QueryTest
       Some(StringType)
   }
 
+  val testH2DialectTinyInt = new JdbcDialect {
+    override def canHandle(url: String): Boolean = url.startsWith("jdbc:h2")
+    override def getCatalystType(sqlType: Int, typeName: String, size: Int, md: MetadataBuilder) :
+    Option[DataType] = {
```

Comment:
Not a big deal but there's style guide https://github.com/databricks/scala-style-guide (see http://spark.apache.org/contributing.html)

Reply:
@HyukjinKwon  Thanks for the review! I have updated the PR after running the ```./dev/scalafmt``` script.